session/session.go

```diff
@@ -189,11 +190,11 @@ func (s *session) handleRestoredCh(pch *pclient.Channel) {
 	if pch.Phase() != pchannel.Acting {
 		return
 	}
-	peers := pch.Peers()
-	parts := make([]perun.Peer, len(peers))
-	aliases := make([]string, len(peers))
+	peerIDs := pch.Peers()
+	parts := make([]perun.PeerID, len(peerIDs))
+	aliases := make([]string, len(peerIDs))
```

Comment:
```suggestion
	parts := make([]perun.PeerID, len(partOffChainAddrs))
	aliases := make([]string, len(partOffChainAddrs))
```

In accordance with the variable rename in the above comment.